scripts/compile/compile-stage.ps1

```diff
@@ -48,15 +48,6 @@ if (!$?) {
     Exit 1
 }
 
-# Build the projects that we are going to ship as nuget packages
-$ProjectsToPack | ForEach-Object {
```

Comment:
dotnet pack runs build on projects we are building and i didn't find `forPackaging` folder references anywhere else.